data/data/coreos/marketplace-rhcos.json

```diff
@@ -0,0 +1,116 @@
+{
+    "aarch64": {
+        "azure": {
+            "no-purchase-plan": {
+                "hyperVGen2": {
+                    "publisher": "azureopenshift",
+                    "offer": "aro4",
+                    "sku": "419-arm",
+                    "version": "419.6.20250523"
+                }
+            }
+        }
+    },
+    "x86_64": {
+        "azure": {
+            "no-purchase-plan": {
+                "hyperVGen1": {
+                    "publisher": "azureopenshift",
+                    "offer": "aro4",
+                    "sku": "aro_419",
+                    "version": "419.6.20250523"
```

Comment:
Hmm, what's up with the version string here? This should just be the same `9.6.xxx` version string inherited from the non-marketplace image.

Reply:
Note that we can't change a version once it's out, only publish new ones.  Also we won't be able to deprecate images until MCO implements skew enforcement and we can confirm no one is running older marketplace images.